.github/workflows/R-CMD-check.yaml

```diff
@@ -104,6 +104,18 @@ jobs:
         run: rcmdcheck::rcmdcheck(args = c("--no-manual", "--as-cran"), error_on = "warning", check_dir = "check")
         shell: Rscript {0}
 
+      - name: Install cmdstan, stan, math from develop
+        run: |
+          cmdstanr::install_cmdstan(cores = 2, cmdstan_branch = "develop",
```

Comment:
Yeah, we initially install cmdstan with the master branch. For these kinds of PRs this is not going to work. I will address that separately.